src/controllers/general/index.ts

```diff
@@ -0,0 +1,33 @@
+import { Request, Response } from 'express';
+import validateBody from '../../middlewares/validateBody';
+import { ObjectType, Setting } from '../../types';
+import database from '../../utils/database';
+import postToSlack from '../../utils/postToSlack';
+
+import { noContent, success, badRequest } from '../../utils/responses';
+import { contactValidator } from '../../validator';
+
+export const status = async (request: Request, response: Response) => {
+  const settings = await database.settings.findMany();
+  return success(
+    response,
+    // Transform [{ id: "x", value: "true"}] to { x: true }
+    settings.reduce((previous: ObjectType, current: Setting) => {
+      Object.assign(previous, { [current.id]: current.value === 'true' });
+      return previous;
+    }, {}),
+  );
+};
+
+export const postContact = [
```

Comment:
```suggestion
export const contact = [
```
pourquoi `postContact` si au dessus c'est `status` ?